person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
- [OPTIONAL] I think it would be a good idea if the id can be generated randomly, it can be similar to 👉🏾  `@id = Random.rand(1..1000)`

Reply:
Hi @cynthiainga. Thank you very much for the feedback and tips. For the first one, I think you mean I used two back-slashes \\, which I should correct.